tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/util/ViewUtil.java

```diff
@@ -0,0 +1,29 @@
+package org.pitest.pitclipse.ui.util;
+
+import org.eclipse.swtbot.eclipse.finder.SWTWorkbenchBot;
+import org.eclipse.swtbot.eclipse.finder.widgets.SWTBotView;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotShell;
+
+/**
+ * 
+ * @author Jonas Kutscha
+ *
+ */
+public class ViewUtil {
+    /**
+     * Opens the view in the given node and returns its handle, identified with the id.
+     * @param bot      used for the menus
+     * @param viewName name of the view seen, in "Show View" shell
+     * @param nodeName name of the node, where the view can be found, in "Show View" shell
+     * @param viewId   the view id which identifies the view
+     * @return the handle of the view
+     */
+    public static SWTBotView openViewById(SWTWorkbenchBot bot, String viewName, String nodeName, String viewId) {
+        bot.menu("Window").menu("Show View").menu("Other...").click();
```

Comment:
If you know the view id, it's better to open a view programmatically, instead of mimicking the operations relying on Eclipse dialogs (in SWTBot tests this might easily become flaky).

For example something like

```java
PlatformUI.getWorkbench().getActiveWorkbenchWindow().getActivePage().showView("viewId");
```

you have to do that inside a UI thread block in SWTBot, similar to what is done in `org.pitest.pitclipse.ui.tests.AbstractPitclipseSWTBotTest.openJavaPerspective()`

then, outside the block, you can get hold on the opened view with SWTBot: `bot.viewById(viewId)`